### PR TITLE
Removes cloudflare dependency from debug.html.ep

### PR DESCRIPTION
**Pull request by hernan604**
### Summary
Removes cloudflare dependency from debug.html.ep and replaces the font awesome icons with svg from:

- https://icones.js.org/collection/fa6-regular

- https://icones.js.org/collection/fa-brands

### Motivation
User complains that debug.html.ep hangs when he is offline and the issue seems to be because of external dependency on icons hosted on cloudflare CDN 

### References
Jun/18/2023 - $user: it might be an improvement to remove the line referencing cloudflare in debug.html.ep
Jun/18/2023 - kraih: then there's no font awesome, feel free to propose an alternative way to load it that doesn't make it hang though

### CODE REVIEW

File: lib/Mojolicious/resources/templates/mojo/debug.html.ep [hunk 1]
```diff
@@ -13,14 +13,57 @@
...
+
+      .fa-linkedin::before {
+        width: 21px;
+        height: 24px;
+        content: "\00a0";
+        display: inline-block;
+        color: #777;
+        line-height: 24px;
+        background: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="gray" d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5c0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7c-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5c67.2 0 79.7 44.3 79.7 101.9V416z"/></svg>') no-repeat;
+
```
Comment (kraih) [inline lib/Mojolicious/resources/templates/mojo/debug.html.ep:63]: The formatting looks inconsistent.

**Review by marcusramberg — DISMISSED** [review]
Nice. Commits need to be squashed, but lgtm :)

**Comment by hernan604** [comment]
Squashed

**Comment by kraih** [comment]
Would be nice if you fixed the same issue in [mojo.js](https://github.com/mojolicious/mojo.js/blob/e95755f6da4b3c6a519c1d4a638130a1816527b4/vendor/views/mojo/debug.html.tmpl#L11) too.

**Review by marcusramberg — APPROVED** [review]
💯

**Review by Grinnz — APPROVED** [review]

**Comment by kraih** [comment]
It appears the mouse over effect no longer works after the patch.

**Comment by hernan604** [comment]
> 

Sent a fix in another PR https://github.com/mojolicious/mojo/pull/2079

**Comment by kraih** [comment]
Note that this PR breaks the license of the contributed code, i have fixed this after the fact. Please be more careful in the future and respect license agreements.

**Pull request merged**